docs/evaluation/how_to_guides/annotation_queues.mdx

```diff
@@ -14,11 +14,28 @@ While you can always [annotate runs inline](./annotate_traces_inline), annotatio
 To create an annotation queue, navigate to the **Annotation queues** section through the homepage or left-hand navigation bar.
 Then click **+ New annotation queue** in the top right corner.
 
-![](./static/annotation_queue_form.png)
+![](./static/create_annotation_queue_new.png)
+
+### Basic Details
 
 Fill in the form with the **name** and **description** of the queue.
 You can also assign a **default dataset** to queue, which will streamline the process of sending the inputs and outputs of certain runs to datasets in your LangSmith workspace.
 
+### Annotation Rubric
+
+First, type some high-level instructions for your annotators, which will be shown in the sidebar on every run.
```

Comment:
the word 'type' messed me up

Maybe "Begin by drafting some high-level"...

Reply:
oops already merged will merge another one, i agree